pkgs/applications/radio/sdrplay/default.nix

```diff
@@ -50,7 +60,7 @@ stdenv.mkDerivation rec {
     ln -s "$out/lib/$libName.so.$majorVersion" "$out/lib/$libName.so"
     cp "${arch}/sdrplay_apiService" $out/bin/
     cp -r inc/* $out/include/
-    cp 66-mirics.rules $out/lib/udev/rules.d/
+    cp -r $udev_rules $out/lib/udev/rules.d/
```

Comment:
As you're creating a derivation already (by using `writeText`), maybe it's enough to link the rules, instead of copying them?

Reply:
tbh, I rather do not want it to be its own derivation at all. I'm new to NixOS so not sure how it is done properly otherwise. I've seen that this is supposed to be ideomatic, so here we are.

Reply:
Just set `udevRules = ''[content]''` on the derivation (or `inherit` it, if you move it in the `let ... in`). Also set `passAsFile = ["udevRules"];` on the derivation, and anywhere in the scripts you have access to `$udevRulesPath`, which you can now `cp` without creating any extra derivations.

Reply:
```suggestion
    ln -s $udev_rules $out/lib/udev/rules.d/66-sdrplay.rules
```